Client/game_sa/CHudSA.cpp

```diff
@@ -171,3 +175,38 @@ void CHudSA::ResetComponentAdjustment()
     MemPut<float>(m_pfCameraCrosshairScale, 192.0f);
     m_fSniperCrosshairScale = 210.0f;
 }
+
+bool CHudSA::IsCrosshairVisible()
```

Comment:
Const? Maybe noexcept?

Reply:
IsHudComponentVisible cannot be const at this moment so this function also